lang/Makefile.common

```diff
@@ -0,0 +1,68 @@
+#
+# Generic makefile to build and test a parser for a specific language.
+#
+
+PROJECT_ROOT = $(shell git rev-parse --show-toplevel)
+DUNE_BIN = $(PROJECT_ROOT)/bin
+NODE_BIN = $(PROJECT_ROOT)/node_modules/.bin
+SCRIPTS = $(PROJECT_ROOT)/scripts
+
+# Build the parsers.
+#
+.PHONY: build
+build:
+	$(MAKE) prep
```

Comment:
very nice names. Useful because it can be quickly become confusing with all those code generators involved.